README.rst

```diff
@@ -48,10 +48,13 @@ To use Mopidy-AlsaMixer the ``audio/mixer`` configuration value must be set to
 The default Mopidy-AlsaMixer configuration will probably work for most use
 cases. If not, the following configuration values are available:
 
-- ``alsamixer/card``: Which soundcard to use, if you have more than one.
-  Numbered from 0 and up. 0 is the default.
+- ``alsamixer/card``: Which soundcard should be used, specified by its index.
+  Numbered from 0 and up. If specified, ``alsamixer/device`` is ignored.
 
-- ``alsamixer/control``: Which ALSA control to use. Defaults to ``Master``.
+- ``alsamixer/device``: Which soundcard should be used, specified by its string
+  alias. Defaults to ``default``.
+
```

Comment:
Maybe move this up top, both here and in `get_config_schema()` as this becomes the primary setting?